config/initializers/statsd.rb

```diff
@@ -91,3 +91,6 @@
 
 # init SentryJob error monitoring
 StatsD.increment(SentryJob::STATSD_ERROR_KEY, 0)
+
+# init Search
+StatsD.increment("#{Search::Service::STATSD_KEY_PREFIX}.exceptions", 0, tags: ['exception:429'])
```

Comment:
@wunderhund - We will either be creating a new graph in grafana to capture this 429 data, or adding this data to an existing one.  This depends on your (or rather DevOps) recommendations. 

Will this bucket and tag name work for either case?

Reply:
@wunderhund - When you have a sec, can we get your thoughts on:  https://github.com/department-of-veterans-affairs/vets-api/pull/2416#discussion_r230423927

Reply:
@hpjaj sorry for the late reply, somehow I missed being tagged in this. 

1. I think we should create a new Grafana dashboard for Search.gov since we don't have one already, and put this in there. 
2. As far as I know? I'm not really a StatsD expert so you'd probably have a better idea than I do.